roles/ceph-rbd-mirror/tasks/start_rbd_mirror.yml

```diff
@@ -1,7 +1,14 @@
 ---
-- name: start and add that the rbd mirror service to the init sequence
+- name: stop and remove the generic rbd-mirror service instance
   service:
       name: "ceph-rbd-mirror@{{ ceph_rbd_mirror_local_user }}"
+      status: stopped
+      enabled: no
+  changed_when: false
```

Comment:
```ignore_error``` isn't valid on a service -- it just ensures the state and enable values match systemd